lib/open_food_network/orders_and_fulfillments_report/customer_totals_report.rb

```diff
@@ -206,6 +211,11 @@ def shipping_method(line_items)
         shipping_rate = shipping_rates.find(&:selected) || shipping_rates.first
         shipping_rate.try(:shipping_method)
       end
+
+      def scoper_for(distributor_id)
+        @scopers_by_distributor_id[distributor_id] ||=
+          OpenFoodNetwork::ScopeVariantToHub.new(distributor_id)
```

Comment:
I'm slightly worried about the potential impact of this, given that `ScopeVariantToHub` fetches all of the hub's overrides in it's initializer, in the context of large hubs with a lot of overrides (France has a few) that are already struggling to load reports...

Reply:
Fair enough. I'm pretty rusty and lots has changed (nice work!), any suggestions or examples of a better approach to this kind of problem?

Reply:
It might be OK. I guess we can just test it in a staging server with lots of data? 

I don't think there's an easy alternative for displaying the alternate SKUs. 

I'm thinking this change might also need some product team input...?

Reply:
Righto, who do I ping? Some acceptance criteria that I can work towards would be really helpful.

Reply:
Maybe @RachL and @lin-d-hop could weigh in on it...

We're changing the SKUs displayed in the customer totals report to show the SKUs from variant overrides instead of the original variant SKU. It could be a big surprise for some users. Maybe it would just need an announcement note in the release?

Reply:
I do agree that it is the technically correct behaviour. But also concerned of the consequences.
SKU is rarely used in the UK so my preference would be this fix.

Would like to hear @RachL's thoughts.

The biggest concern would be performance implications. @Matt-Yorkley can you help us to be as certain as possible this won't increase any server explosion rates?

Reply:
hmmmm I have no idea if people use SKU on my instance... but if they do I think the change is logical. So I guess we are good to go 👍